templates/velero.yaml

```diff
@@ -1,3 +1,59 @@
+# ------------------------------------------------------------------------------
```

Comment:
Please, let's simply add a link to the Velero helm, this is making this YAML hard to read and maintain :/.

Reply:
> Let's remove all that commented code in the yaml please

Why?

> Question: What is the size of the minio volume?

[10g](https://github.com/kubeaddons/extrasteps/blob/master/pkg/minio/minio.go#L164) per instance (default 4 instances)

> Please, let's simply add a link to the Velero helm, this is making this YAML hard to read and maintain :/.

I'm confused by this assertion. Can you help me to better understand your take on this please?

Reply:
i'd prefer a link to, but it's a non-blocker.

Reply:
What is the value of adding all these text ? In my opinion, the value could be provided by a link to the velero helm and minio operator. There you can add the README and additional explanations about this addon. Let's keep this file as short as possible.

YOu could add this text to the mesosphere/chart of velero.

Reply:
Otherwise we could add a docs section here for velero. However I think that a link would be helpful enough.

Reply:
I must respectfully disagree with your position on this, however since this is a blocker and holding up other work I've made a compromise and removed the testing section which included hints about how to deploy the addon:

a25e1eaa83c79961b87dacc5d6b1eaaa48b0e82b

I've also created a JIRA which covers the need to testing tooling for addons such as this was attempting to provide a little bit of for velero:

[DCOS-56456](https://jira.mesosphere.com/browse/DCOS-56456)

Let me know if this is an acceptable compromise from your end. I will mark this as resolved, please mark it as unresolved again if there is more you feel needs to be done.

Reply:
I'm good if hector is ok. I still prefer a link.

Reply:
What link @s12chung? There are three links below did you want something else added or did you mean JUST a link?